### ISSUE TITLE
Genetics UI modification borked

### ISSUE THREAD

**Issue by M3hillus** [ParadiseSS13/Paradise#1741]
In a recent round I was reviving a gibbed person, via the good old monkeyman braintransplant trick. Used strange reagent to revive them post surgery to skip the whole cloning thing.
Looking at the paitent, I realised they still had the name Monkey(659). 
I took the subject to genetics, and after much fiddling, was unable to change their name. Appearance and sex, yes, but not their name.

**Comment by TheDZD** [comment]
Were you changing their unique identifiers (UI) or their unique enzymes (UE) when trying to change their name? UI only affect appearance, while the UE affect the character name.

**Comment by M3hillus** [comment]
After actually using the console myself last-night, reviving a husked player via monkeyman bodyswap, the UI values for a good portion of the blocks are locked to null. I'm assuming if I did a headswap it might have gone differently, but I'm not sure how mob identity works in that regard. If this is not a bug, and the UI stuff was removed because of some reason during my break from the server, that's fine. But even as rare as it is, it does affect gameplay, at least mildly at the rp level. I'd rather be John doe than a human named Monkey (453).
